src/legacyTheme.js

```diff
@@ -71,8 +71,6 @@ export const colors = {
   orange3
 }
 
-const legacyTheme = Object.assign({}, theme, {
-  colors
-})
+const legacyTheme = {...theme, colors: {...colors}}
```

Comment:
This can be written as `{ ...theme, { colors } }` which would be one less call to Babel's `extends` function